src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -2797,9 +2797,20 @@ namespace SurfaceGeometry {
                              std::string{RoutineName} + "When using DElight daylighting the presence of exterior shading surfaces is ignored.");
         }
 
-        // Initialize run time surface arrays
         for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; SurfNum++) {
+            // Initialize run time surface arrays
             state.dataSurface->SurfActiveConstruction(SurfNum) = state.dataSurface->Surface(SurfNum).Construction;
+            state.dataSurface->Surface(SurfNum).RepresentativeCalcSurfNum = SurfNum;
+            // Automatic Surface Multipliers: Assign representative heat transfer surfaces
+            if (state.dataSurface->UseRepresentativeSurfaceCalculations && state.dataSurface->Surface(SurfNum).HeatTransSurf &&
+                state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, "ZoneProperty:UserViewFactors:BySurfaceName") == 0) {
```

Comment:
Probably shouldn't call this for every surface. Call once and set a flag. Better yet, split this loop into two and only do the second part if `UseRepresentativeSurfaceCalculations` is true and there are no user defined view factors.